ex1/index.html

```diff
@@ -12,10 +12,18 @@ <h1><u>The blue button</u></h1>
     <h2>This is my blue button</h2>
     <button type="button"
     class="button"
-    onclick="changeColor()"
-    id="demo">
+    onclick="handleButtonAndImageStyles()"
+    id="buttonID">
```

Comment:
real bad name for a button id.  please think of what this button does or represents and change the id name accordingly.  Another thing: in css convention, there's no camelCase.  So in this case it would have been button-id.

While you're at it, please update the name of the class center and change it from class to id.  call it something general the defines the whole block that's under it in the HTML.